front-end/src/app/shared/models/transaction-types/FEDERAL_ELECTION_ACTIVITY_VOID.model.ts

```diff
@@ -0,0 +1,26 @@
+import { LabelUtils } from 'app/shared/utils/label.utils';
+import { schema } from 'fecfile-validate/fecfile_validate_js/dist/DISBURSEMENTS_FEA';
+import { SchBTransactionType } from '../schb-transaction-type.model';
+import { SchBTransaction, ScheduleBTransactionTypeLabels, ScheduleBTransactionTypes } from '../schb-transaction.model';
+import { TransactionGroupO } from '../transaction-groups/transaction-group-o.model';
+import { STANDARD_CONTROLS, TransactionNavigationControls } from '../transaction-navigation-controls.model';
+import { PurposeDescriptionLabelSuffix } from '../transaction-type.model';
+import { AggregationGroups } from '../transaction.model';
+
+export class FEDERAL_ELECTION_ACTIVITY_VOID extends SchBTransactionType {
+  transactionGroup = new TransactionGroupO();
+  title = LabelUtils.get(ScheduleBTransactionTypeLabels, ScheduleBTransactionTypes.FEDERAL_ELECTION_ACTIVITY_VOID);
+  schema = schema;
+  override negativeAmountValueOnly = true;
+  override showAggregate = false;
+  override purposeDescriptionLabelSuffix = PurposeDescriptionLabelSuffix.REQUIRED;
+  override navigationControls: TransactionNavigationControls = STANDARD_CONTROLS;
```

Comment:
should we include 
```
override isRefundAggregate = true;
```

Reply:
@toddlees I think this may not be needed since showAggregate is false and isRefundAggregate is only used inside this if statement https://github.com/fecgov/fecfile-web-app/blob/99e3d7078cf3fb5c1f939655057fc37e682cb1be/front-end/src/app/shared/components/transaction-type-base/transaction-form.utils.ts#L75

Should we set it in case we decide to display the aggregate?

Reply:
ah i see. i think you're right to not include it.  If the property was called isRefund, i think i'd say we should add it, but as it's named, it only pertains to the aggregate which is not shown